robot/Cumulus/resources/CustomSettingsPageObject.py

```diff
@@ -0,0 +1,37 @@
+import time
+from cumulusci.robotframework.utils import capture_screenshot_on_error
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import pageobject
+from BaseObjects import BaseNPSPPage
+from NPSP import npsp_lex_locators
+from logging import exception
+
+@pageobject("Custom", "CustomSettings")
+class CustomSettingsPage(BaseNPSPPage, BasePage):
+
+    
+    def _go_to_page(self, filter_name=None):
+        """Go to Custom Settings page"""
+        url = self.cumulusci.org.lightning_base_url
+        url = "{}/lightning/setup/CustomSettings/home".format(url)
+        self.selenium.go_to(url)
+        self.salesforce.wait_until_loading_is_complete()
+        self.npsp.wait_for_locator("frame_new","Custom Settings","Custom Settings")
+        
+    def select_settings_option(self,setting_name,value): 
+        """selects frame with setting_name and clicks on the option 
+           identified by value for the given setting and unselects frame"""
+        self.npsp.choose_frame("Custom Settings")
+        locator = npsp_lex_locators['custom_settings']['link'].format(setting_name,value)
+        self.selenium.wait_until_element_is_visible(locator, timeout=60)
+        self.selenium.scroll_element_into_view(locator)
+        self.selenium.click_element(locator)
+        self.selenium.unselect_frame()
+        self.salesforce.wait_until_loading_is_complete()
+           
+    def verify_current_page(self,page_name):
```

Comment:
The keyword name is not matching the description mentioned:" verify page contains page_name and select frame identified with page_name". Please change it to be something like "verify_and_select_frame"

Reply:
since the operation of selecting a frame is something that should be invisible, I named the keyword as verify current page but gave the details in the description so that anyone who wants to reuse this keyword can read and understand what it can do. Do you think this will cause confusion?

Reply:
As an end user it was not quite evident when I wanted to use the keyword. Feel changing the keyword would help.

Reply:
ok will change to verify page and select frame